testutil/singlevm/setup.sh

```diff
@@ -146,6 +147,8 @@ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout controller_key.pem -
 #Copy the certs
 sudo cp -f controller_key.pem /etc/pki/ciao
 sudo cp -f controller_cert.pem /etc/pki/ciao
+sudo ln -s /etc/pki/ciao/controller_key.pem /etc/pki/ciao/ciao-image-key.pem
+sudo ln -s /etc/pki/ciao/controller_cert.pem /etc/pki/ciao/ciao-image-cacert.pem
```

Comment:
Well @rbradford and I discussed this and we decided that it wasn't worth the extra effort of creating an extra certificate.  Does it really buy us anything in Single VM to use separate certs?